wrapper/src/main/java/software/amazon/jdbc/ConnectionPluginChainBuilder.java

```diff
@@ -101,6 +103,7 @@ public class ConnectionPluginChainBuilder {
           put(IamAuthConnectionPluginFactory.class, 1000);
           put(AwsSecretsManagerConnectionPluginFactory.class, 1100);
           put(FederatedAuthPluginFactory.class, 1200);
+          put(LimitlessConnectionPluginFactory.class, 1250);
```

Comment:
I'd put it before all authentication plugins (IAM and others). In theory, IAM token should be generated based on node name that gets available in Limitless plugin.